Test/ArpTest.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using NUnit.Framework;
+using SharpPcap;
+using SharpPcap.LibPcap;
+
+namespace Test
+{
+    [TestFixture]
+    public class ArpTest
+    {
+        [Test]
+        public void TestArp()
+        {
+            var d = TestHelper.GetPcapDevice();
+            var arp = new ARP(d);
+
+            // timeout should not be null
+            Assert.IsNotNull(arp.Timeout);
+
+            // and we can set a timeout
+            arp.Timeout = new TimeSpan(0, 0, 2);
+
+            var localMAC = d.MacAddress;
+
+            if ((localMAC != null) && (localMAC.GetAddressBytes().Length != 0))
+            {
+                var destinationIP = new System.Net.IPAddress(new byte[] { 8, 8, 8, 8 });
+                try
+                {
+                    var physicalAddress = arp.Resolve(destinationIP);
+                }
+                catch (PcapException ex)
+                {
+                    // Likely caused by a 'Can't send packet: send: Network is down'
+                    Assert.Ignore(ex.Message);
```

Comment:
Should not happen anymore since you are using `TestHelper.GetPcapDevice`

Reply:
k, checking now with CI